src/libraries/System.Collections.NonGeneric/tests/SortedListTests.cs

```diff
@@ -266,7 +266,7 @@ public void EnsureCapacity_NewCapacityLessThanMin_CapsToArrayMaxLength()
             const int MinCapacity = InitialCapacity * 2 + 1;
             var sortedList = new SortedList(InitialCapacity);
 
-            MethodInfo ensureCapacity = sortedList.GetType().GetMethod("EnsureCapacity", BindingFlags.NonPublic | BindingFlags.Instance);
+            MethodInfo ensureCapacity = typeof(SortedList).GetMethod("EnsureCapacity", BindingFlags.NonPublic | BindingFlags.Instance);
```

Comment:
Does this change have any impact on trimmability? The test method already has a static dependency on the type.

Reply:
It fixes trimmability of the test. Trimming doesn't track types of things stored in the locals (it could... but it doesn't). SortedList type is not sealed, so trimming doesn't know what the actual type was we're looking up the method on. Replacing the `GetType` call with a `typeof` makes it possible to statically analyze this. This generates a trimming warning no matter if the method is rooted from elsewhere.

Trimming is free to optimize used but not reflected on methods in any suitable way (they can be inlined, their names stripped, etc.). So the existing reflection didn't actually work with trimming.